CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # CHANGE LOG
+## v4.6.0
+
+* **NEW** Added `duitnow_obw sourceType to enable charge creation for duitnow` source types (#156)
```

Comment:
I think the description should not be in the code highlight. Can you rephrase this?

Reply:
updated @nuxzero